ui/src/organizations/components/UpdateBucketOverlay.tsx

```diff
@@ -104,8 +104,10 @@ export default class BucketOverlay extends PureComponent<Props, State> {
     this.setState({bucket})
   }
 
-  private handleChangeRuleType = ruleType => {
-    this.setState({ruleType})
+  private handleChangeRuleType = async (
+    ruleType: BucketRetentionRules.TypeEnum
+  ) => {
+    await this.setState({ruleType})
```

Comment:
I'm not clear what this `await` is helping solve. Should `onChangeRuleType` be awaited in `handleRadioClick` or is it being awaited elsewhere?

Reply:
Mistake on my part, it is not needed anymore